WordPress/openverse
# Add consent decision-making process documentation

## Decision-making process

For the purposes of this discussion, we will follow [Openverse's decision-making model](https://wordpress.github.io/openverse/reference/consent_decision_making.html). This process follows formalised steps with specific expectations of participants. Before contributing, please read the document added in this PR as well as [Openverse's Code of Conduct](https://github.com/WordPress/openverse/blob/main/CODE_OF_CONDUCT.md).

The consent decision-making document linked above also includes instructions for opting out of a decision discussion you do not wish to or cannot participate in.

## Current round

The discussion is currently in the **Decision** round.

> **Note**
> 
> I modified the above text to remove the link to "this document" that doesn't exist yet (because we haven't merged this yet). For now it just refers to the proposal text in the PR.


## Fixes




Fixes #874 by @sarayourfriend 

## Description




This PR introduces the public facing document describing the Openverse consent decision-making process.

## Testing Instructions


Read the new document. Ensure that the revisions agreed upon in previous discussions about this process are adequately captured. Likewise, ensure that the document sufficiently covers the basic requirements of the process with an eye for accessibility.

If reviewers have any suggestions for ways to improve the accessibility of the document for a wider audience, please let me know.

## Checklist



      `Update index.md`).
      a parent feature branch.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin



<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
